src/IntegerPicker/IntegerPicker.js

```diff
@@ -247,7 +247,7 @@ module.exports = kind(
 	*/
 	create: function () {
 		Control.prototype.create.apply(this, arguments);
-		this.verifyValue();
+		this.rangeChanged();
```

Comment:
Calling this in `create` causes the `onChange` event to be fired during create. Which is something we REALLY don't want. It was being avoided in the current implementation and may cause problems by changing this.